pipes-schema-validation-confluent/README.md

```diff
@@ -0,0 +1,240 @@
+# Event schema validation for Apache Kafka with EventBridge Pipes and Confluent Schema Registry
+
+This pattern allows you to perform event schema validation and transformation from Avro to JSON for events consumed by [Amazon EventBridge Pipes](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-pipes.html) from a [self managed Apache Kafka stream as source](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-pipes-kafka.html) using [Confluent Schema Registry](https://gitlab.aws.dev/pbv/pipes-schema-validation).
```

Comment:
I recommend mentioning that this pattern also uses Confluent Cloud. Currently it reads as self-managed but with CC schema registry - the self-managed part is only because that's how you configure CC. It should be made more clear in the intro that this is based on a CC cluster